tests/Unit/Evolution/Systems/Cce/Actions/Test_SendNextTimeToCce.cpp

```diff
@@ -0,0 +1,205 @@
+        &runner, 0_st,
+        {std::make_unique<Cce::InterfaceManagers::GhLocalTimeStepping>(3_st)});
+
+    // Emplace elements.
+    for (const auto& element_id : element_ids) {
+      // 1. Get mesh
+      auto mesh =
+          get<1>(InterpolateOnElementTestHelpers::make_volume_data_and_mesh(
+              domain_creator, domain, element_id));
+
+      // 2. emplace element.
+      ActionTesting::emplace_component_and_initialize<elem_component>(
+          &runner, element_id,
+          {time_step_id_, next_time_step_id_, time_step_, mesh,
+           interp_point_info});
+    }
+    ActionTesting::set_phase(make_not_null(&runner),
+                             test_metavariables::Phase::Testing);
+
+    // Call the action on all the elements.
+    for (const auto& element_id : element_ids) {
+      ActionTesting::next_action<elem_component>(make_not_null(&runner),
+                                                 element_id);
+    }
+    if (not is_substep_) {
+      // run the (single) simple receive action on the boundary
+      runner.template invoke_queued_simple_action<
+          mock_gh_worldtube_boundary<test_metavariables>>(0_st);
+
+      CHECK(next_time_received == expected_next_time_step_);
+    }
+
+    // check that there aren't any more simple actions queued
+    CHECK(ActionTesting::is_simple_action_queue_empty<
+          mock_gh_worldtube_boundary<test_metavariables>>(runner, 0));
+  }
+
+ private:
+  TimeStepId time_step_id_;
+  TimeStepId next_time_step_id_;
+  TimeDelta time_step_;
+  TimeStepId expected_next_time_step_;
+  bool is_substep_;
+};
+
+void test_send_time_to_cce(const bool substep) noexcept {
+  Parallel::register_derived_classes_with_charm<
+      Cce::InterfaceManagers::GhInterfaceManager>();
+  using metavars = test_metavariables;
+  using elem_component = mock_element<metavars>;
+  TimeStepId time_step_id;
+  TimeStepId next_time_step_id;
+  TimeStepId expected_next_step;
+  if (substep) {
+    time_step_id = TimeStepId{
+        true, 0_st, {{0.0, 0.1}, {1, 2}}, 1_st, {{0.0, 0.1}, {1, 1}}};
+    next_time_step_id = TimeStepId{
+        true, 0_st, {{0.0, 0.1}, {1, 2}}, 2_st, {{0.0, 0.1}, {3, 4}}};
+  } else {
+    time_step_id = TimeStepId{true, 0_st, {{0.0, 0.1}, {1, 2}}};
+    next_time_step_id = TimeStepId{
+        true, 0_st, {{0.0, 0.1}, {1, 2}}, 1_st, {{0.0, 0.1}, {1, 1}}};
+    expected_next_step = TimeStepId{true, 1_st, {{0.1, 0.2}, {0, 2}}};
+  }
+  const TimeDelta time_step{{0.0, 0.1}, {1, 2}};
+  InterpolateOnElementTestHelpers::test_interpolate_on_element<metavars,
+                                                               elem_component>(
+      initialize_elements_and_queue_simple_actions<metavars, elem_component>{
+          time_step_id, next_time_step_id, time_step, expected_next_step,
+          substep},
+      std::make_unique<::TimeSteppers::RungeKutta3>());
+
+  // check that the 'next' time made it into the interpolation manager
```

Comment:
Is this an extra comment, or a missing check below the comment?

Reply:
oops, yep, this is an extra comment before I decided to put all the checks into the previous function.